README.adoc

```diff
@@ -21,6 +21,7 @@ toc::[]
 
 * https://calcite.apache.org/[Apache Calcite] - SQL parser, building blocks for datastores.
 * https://druid.apache.org/[Apache Druid] - A high performance real-time analytics database.
+* https://www.influxdata.com/[InfuxDB] - Purpose-Built Open Source Time Series Database.
```

Comment:
InfuxDB -> InfluxDB

Could you also move it after Apache Pinot, so to keep it alphabetically sorted? Thanks!

Reply:
Oops! Sure thing

Reply:
Done